mis_builder/views/mis_report_instance.xml

```diff
@@ -380,8 +380,8 @@
                                 invisible="type != 'date_range'"
                                 required="type == 'date_range' and mode == 'relative'"
                             />
-                            <field name="offset" />
-                            <field name="duration" />
+                            <field name="offset" string="Offset (Quarters)" attrs="{'invisible': [('type', '!=', 'q')]}"/>
+                            <field name="duration" string="Duration (Quarters)" attrs="{'invisible': [('type', '!=', 'q')]}"/>
```

Comment:
This looks like a breaking change - why are people not allowed to see the offset and duration in other modes anymore? Eg for month offset and duration are also used, but now it would become invisible.

Reply:
Not solved?